ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3407,16 +3407,21 @@ signal_list_t* create_operation_node(ast_node_t* op, signal_list_t** input_lists
             break;
         case SR:  // >>
         case ASR: // >>>
-            /* Shifts doesn't matter about port size, but second input needs to be a number */
```

Comment:
let shift rights have the input port width it wants since it cannot grow exponentially with the right-hand side operand but is directly driven by the width of the input.

Reply:
this has still not been addressed.